src/validations/javascript/jshint.js

```diff
@@ -157,6 +169,13 @@ class JsHintValidator extends Validator {
     });
   }
 
+  _getConfig(analyzer) {
```

Comment:
Wasn't quite sure what you meant - I copied into local variables versus referring to the analyzer directly.